backend/src/services/CampusHousingService.ts

```
@@ -0,0 +1,71 @@
+  monthlyRentUSD: number;
+  depositUSD: number;
+  roomType: string;
+  posterName: string;
+  isUtilitiesIncluded: boolean;
+  status: string;
+}
+
+export class CampusHousingService {
+  private sublets: SubletDTO[] = [
+    {
+      id: 'sub-601',
```

ESLint

[error] Replace `'sub-601'` with `"sub-601"`

(prettier/prettier)